torch/csrc/autograd/python_variable.cpp

```diff
@@ -2501,10 +2503,35 @@ c10::Layout concrete_layout_fn(
   return toLayout(out.ptr());
 }
 
+c10::SymInt concrete_return_sym_fn(
+    const c10::TensorImpl* self,
+    const char* func_name) {
+  pybind11::gil_scoped_acquire gil;
+  at::impl::MaybeSetTLSOnEntryGuard guard;
+  auto out = torchDispatchFromTensorImpl(
+      self,
+      func_name,
+      py::module::import("torch")
+          .attr("ops")
+          .attr("aten")
+          .attr(func_name)
+          .attr("default")
+          .ptr(),
+      "torch.ops.aten");
+
+  if (out == Py_None) {
+    TORCH_CHECK(
+        !self->has_symbolic_sizes_strides(),
```

Comment:
Why this check? This is a sym_* function.
In the others above, `concrete_sizes_fn` has this check while `concrete_sym_sizes_fn`.